blocklylib-core/src/main/java/com/google/blockly/android/ui/VirtualWorkspaceView.java

```diff
@@ -52,6 +52,7 @@
     private static final int INIT_ZOOM_SCALES_INDEX = 2;
 
     protected boolean mScrollable = true;
+    protected boolean mScalable;
```

Comment:
we are building app for <4 years kids, where pinch create issue, they accidentally zoom in/out, but they are comfortable with pan/scroll